schemas/session-replay/common/pointer-interaction-data-schema.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "session-replay/common/pointer-interaction-data-schema.json",
+  "title": "PointerInteractionData",
+  "type": "object",
+  "description": "Schema of a PointerInteractionData.",
+  "allOf": [
+    {
+      "required": ["source"],
+      "properties": {
+        "source": {
+          "type": "integer",
+          "const": 9,
```

Comment:
Set to `9` to bump the highest source from `schemas/session-replay/browser/incremental-data-schema.json`